test/jaas-test/src/main/resources/suite/defaults.conf.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```

Comment:
propose to remove all declaimers. if necessary we can auto generate them in the whole jagger project